roboragi/Anilist.py

```diff
@@ -166,14 +169,14 @@ def map_media_status(media_status):
 
 def getSynonyms(request):
     synonyms = []
-    synonyms.extend(request['synonyms']) if request['synonyms'] else None
+    synonyms.extend(request.get('synonyms'))
```

Comment:
.get() makes things _so_ much easier in so many places haha. Wish I'd known about it earlier.